### PR TITLE
Code quality CI

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Added a grep for logical operators not being spaced

Went through search and fixed it

Regex I used is 
`(\)|\w)(!=|==|<=|>=|&&|\|\|)` (letter or parenthesis)-operator
`(!=|==|<=|>=|&&|\|\|)(\(|\w)` vice-versa
` (!=|==|<=|>=|&&|\|\|)(\"|')` No space after operator and it's not a string
`(\"|')(!=|==|<=|>=|&&|\|\|) ` vice-versa

## Why It's Good For The Game
Automating style is not only good for the repo but it also greatly appeases my ocd

## Changelog
N/A No player facing

### CODE REVIEW

File: code/modules/antagonists/abductor/abductor.dm [hunk 1]
```diff
@@ -148,7 +148,7 @@
 	var/mob/living/carbon/human/H = owner.current
 	var/gear = tgui_alert(admin,"Agent or Scientist Gear", "Gear", list("Agent", "Scientist"))
 	if(gear)
-		if(gear=="Agent")
+		if(gear =="Agent")
```
Comment (Mothblocks) [inline code/modules/antagonists/abductor/abductor.dm:151]: Looks like this one is still broken

Comment (jlsnow301) [reply code/modules/antagonists/abductor/abductor.dm:151]: hate to keep tacking on more exceptions. ill look for a cleaner way to do it

Comment (Mothblocks) [reply code/modules/antagonists/abductor/abductor.dm:151]: There's no clean way to do it as long as we are using greps and not a sane AST based system, don't bother

Comment (jlsnow301) [reply code/modules/antagonists/abductor/abductor.dm]: added a few lines to catch this case (and where it's not a string ie "==")

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 2]
```diff
@@ -40,10 +40,10 @@
...
 ||||||||||||||||[GAS DATUMS]||||||||||||||||
 ||||||||||||||||\__________/||||||||||||||||
-||||These should never be instantiated. ||||
-||||They exist only to make it easier   ||||
-||||to add a new gas. They are accessed ||||
-||||only by meta_gas_list().            ||||
+|||| These should never be instantiated. ||||
+|||| They exist only to make it easier   ||||
+|||| to add a new gas. They are accessed ||||
+|||| only by meta_gas_list().            ||||
```
Comment (jlsnow301) [inline code/modules/atmospherics/gasmixtures/gas_types.dm:46]: wrecked

**Comment by jlsnow301** [comment]
Screenshot_20230108_021249

**Pull request closed without merging**